src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -5,6 +5,8 @@
 from __future__ import print_function
 
 from collections import Counter, OrderedDict
+import mock
```

Comment:
Not in time for Ignite. This is tracked in #4392. It will involve some potentially sensitive changes to autorest Python and msrestazure.

Reply:
You'll need to include mock as a dep. in setup.py of probably azure-cli-network. See the setup.py of azure-cli-sf.
Right now, it'll work as `mock` is required for azure-cli-sf but that module will be moved soon then all of a sudden this will no longer work.

Reply:
Great catch! Thanks!